app.config.js

```diff
@@ -1,13 +1,17 @@
-import { sentryApi } from './src/config/secrets';
+import { secrets } from './src/config/secrets';
+
+let hooks = secrets.sentryApi?.config
```

Comment:
`let`? `const` here as well, right? or what was your thought?

Reply:
`namespace` of `secrets` missing here?

Reply:
Great catch 👏 

Fixed it to also take the namespace into account now and double checked that it is working as expected.